great_expectations/validator/validation_graph.py

```diff
@@ -367,9 +367,14 @@ def __init__(
         self,
         execution_engine: ExecutionEngine,
         configuration: ExpectationConfiguration,
+        graph: Optional[ValidationGraph] = None,
```

Comment:
Is `ExpectationValidationGraph` part of our public API? I'm wondering if we can make this required instead of `Optional`. I see this saves us some code in validator.py and I like that. I find more optional arguments make classes harder to use. So I'm hoping we can make this required and get both.

Reply:
@billdirks The requirement to always supply non-null `ValidationGraph` has been incorporated.  Thanks!